projects/wave-core-new/src/lib/operators/dialogs/operator-list/operator-list.component.ts

```diff
@@ -128,11 +128,14 @@ export class OperatorListComponent implements OnInit {
             },
             description: 'Get statistics for raster layer'
         },
-        //     {
-        //         component: TimePlotComponent,
-        //         type: TimePlotType,
-        //         description: 'Plot time data'
-        //     }
+        {
+            component: TemporalRasterMeanPlotDialogComponent,
+            type: {
+                NAME: 'Temporal Raster Mean Plot',
```

Comment:
cf. my name dicsussion in the backend PR

Reply:
20d4db5